eng/pipelines/templates/steps/publish-vscode.yml

```diff
@@ -76,3 +82,15 @@ steps:
       displayName: Create GitHub Release and upload artifacts
       env:
         GH_TOKEN: $(azuresdk-github-pat)
+
+  - ${{ if eq('true', parameters.PublishToMarketplace) }}:
+    - task: AzureCLI@2
+      displayName: Publish (using vsce)
+      inputs:
+        azureSubscription: azure-sdk-vsmarketplace
+        scriptType: pscore
+        scriptLocation: inlineScript
+        workingDirectory: release
+        inlineScript: |
+          npm install -g @vscode/vsce
+          vsce publish --azure-credential
```

Comment:
This portion cannot be tested without publishing the extension live. I'm trying to get confirmation that it'll work from someone with knowledge of the `vsce` tool.

Reply:
To get this working: We need to specify the file paths for package, manifest, and signature.